lib/ecto/postgres/pg_rational_migrations.ex

```diff
@@ -0,0 +1,132 @@
+use Vtc.Ecto.Postgres.Utils
+
+defpgmodule Vtc.Ecto.Postgres.PgRational.Migrations do
+  @moduledoc """
+  Migrations for adding rational types, functions and constraints to a
+  Postgres database.
+  """
+  alias Ecto.Migration
+
+  require Ecto.Migration
+
+  @doc section: :migrations_full
+  @doc """
+  Adds raw SQL queries to a migration for creating the database types, associated
+  functions, casts, operators, and operator families.
+
+  Includes all migraitons under the
+  [PgTypes](Vtc.Ecto.Postgres.PgRational.Migrations.html#pgtypes) and
+  [PgFunctions](Vtc.Ecto.Postgres.PgRational.Migrations.html#pgfunctions)
+  headings.
+
+  Safe to run multiple times when new functionality is added in updates to this library.
+  Existing values will be skipped.
+
+  ## Types Created
+
+  Calling this macro creates the following type definitions:
+
+  ```sql
+  CREATE TYPE public.rational AS (
+    numerator bigint,
+    denominator bigint
+  );
+  ```
+
+  ## Schemas Creates
```

Comment:
these sections can be removed from this pr (and can be re-introduced in a follow-up)